src/utils.ts

```diff
@@ -14,19 +15,7 @@ import {
   Token
 } from './types';
 
-const fsReadFile = (fileName: string, encoding: string): Promise<string> => {
-  const readPromise: Promise<string> = new Promise((resolve, reject) => {
-    fs.readFile(fileName, encoding, (error, data) => {
-      if (error) {
-        reject(error);
-        return;
-      }
-      resolve(data.toString());
-    });
-  });
-
-  return readPromise;
-};
+const fsReadFile = promisify(readFile);
```

Comment:
Looks like we can use as "promised" version from node directly here: https://remarkablemark.org/blog/2020/12/20/nodejs-fs-readfile-promise/

Maybe wen can get rid of the `promisify` in that case in general?

Reply:
This is not compatible with node 12 - see last commit